DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift

```diff
@@ -0,0 +1,56 @@
+import SwiftUI
+
+protocol AddGroupNameViewDelegate {
+    func addGroupNameViewDidTapContinue(_ view: AddGroupNameView)
+}
+
+struct AddGroupNameView: View {
+    @State private var groupName: String = ""
+    var delegate: AddGroupNameViewDelegate
+
+    var body: some View {
+        VStack {
+            HStack {
+                Text(AppStrings.AddGroupName.headerTitle.localized())
+                    .font(.custom("IBMPlexSans-SemiBold", size: 23))
+                    .padding(.top, 10)
+
+                Spacer()
+            }
+            .padding(.horizontal)
+
+            VStack(alignment: .leading) {
+                Text(AppStrings.AddGroupName.textFieldTitle.localized())
+                    .foregroundColor(Color.darkText)
+                    .font(.custom("IBMPlexSans-Regular", size: 14))
+
+                TextField(AppStrings.AddGroupName.textFieldPlaceholder.localized(), text: $groupName)
+                    .textFieldStyle(.roundedBorder)
+            }
+            .padding()
+
+            Spacer()
+
+            VStack {
+                Button(action: {
+                    delegate.addGroupNameViewDidTapContinue(self)
```

Comment:
Let's add a rule to make sure we have a group name set before allowing to move to the next step.
1) continue with no text - display alert - Denumirea grupului este obligatorie.
2) continue with text - all good

Reply:
Ok